schemas/Manual.categories.schema.json

```diff
@@ -30,6 +30,10 @@
                             "minItems": 1,
                             "uniqueItems": true
                         },
+                        "yaml_description": {
+                            "description": "(Optional) Dictionary of option-description pairs to add descriptor text to the yaml options defined in 'yaml_option'",
+                            "type": "object"
+                        }
```

Comment:
personally I would define the object using paternproperties but this should work as is

Reply:
Yeah I've never used schemas so I did the bare minimum here. Feel free to suggest whatever makes sense to change here.

How do patternproperties work?

Reply:
Basicaly normaly with an object you can define its properties using `properties` but, when they are define by the user you can use a regex aka a pattern using `patternProperties` 
you could look at [items.schema.json](https://github.com/ManualForArchipelago/Manual/blob/main/schemas/Manual.items.schema.json#L77-L90) for a pretty good example for your situation